src/hotspot/os/linux/gc/z/zMountPoint_linux.cpp

```diff
@@ -28,6 +28,7 @@
 #include "gc/z/zMountPoint_linux.hpp"
 #include "runtime/globals.hpp"
 #include "runtime/os.hpp"
+#include "utilities/globalDefinitions.hpp"
```

Comment:
Did you add this by intention or was it added by your IDE?

Reply:
No, that's fine. My VS code often adds it, but it's not strictly required.

Reply:
Needed for Kim's ALLOW_xxx macros.